plugin/src/App/Components/EditableImage.lua

```diff
@@ -12,15 +12,17 @@ function EditableImage:init()
 end
 
 function EditableImage:writePixels()
-	local image = self.ref.current
+	local image = self.ref.current :: EditableImage
+	local props = self.props
+
 	if not image then
 		return
 	end
-	if not self.props.pixels then
+	if not props.pixels then
 		return
 	end
 
-	image:WritePixels(Vector2.zero, self.props.size, self.props.pixels)
+	image:WritePixelsBuffer(Vector2.zero, props.size, props.pixels)
```

Comment:
Most of these changes are superfluous. I appreciate the initiative for optimization, but creating images isn't a bottleneck to the point where we need to optimize table indexes to this extent.

Would rather that this was just changing `WritePixels` to `WritePixelsBuffer`.